bin/workflows/pycbc_create_offline_search_workflow

```diff
@@ -435,13 +441,13 @@ layout.two_column_layout(rdir['coincident_triggers'], closed_page)
 # run minifollowups on the output of the loudest events
 mfup_dir = rdir['open_box_result/loudest_events_followup']
```

Comment:
mfup dir only used once so is waste of a variable

Reply:
This was actually done to avoid line wrapping problems. If you don't use this things don't fit in 80 characters (and I do like my 80 character vi windows!)

Reply:
I was considering line https://github.com/gwastro/pycbc/pull/3193/files/faa728a262b422f90c440e7b85cba54b7b6b65ed#diff-7e386e586cbba96a2655d0b5293e2845R452 and how it is done there

Reply:
Yeah, but that line is then longer than 80 characters, which is not PEP-8 (and looks bad on my vi terminal windows). I had to fix line 446 as well now!